customizable.html

```diff
@@ -126,7 +126,23 @@ <h2 id="resources">Learn more</h2>
             <li><a href="https://www.w3.org/WAI/intro/people-use-web/stories.html#classroomstudent">Ms. Olsen, Classroom student with attention deficit hyperactivity disorder (ADHD) and dyslexia</a></li>
           </ul>
         </li>
-        <li><strong>WCAG 2.0:</strong> <a href="https://www.w3.org/WAI/WCAG20/quickref/?tags=text">Success Criteria relating to "text"</a></li>
+        <li><strong>User Needs:</strong>
+          <ul>
+            <li><a href="http://www.w3.org/TR/low-vision-needs/"> Accessibility Requirements for People with Low Vision            </a></li>
+          </ul>
+        </li>
+        <li><strong>WCAG 2.0 </strong>(<a href="https://www.w3.org/WAI/intro/wcag">Web Content Accessibility Guidelines Overview</a>):
+          <ul>
+            <li><a href="https://www.w3.org/WAI/WCAG20/quickref/?tags=text">Success Criteria relating to "text"</a></li>
+          </ul>
+        </li>
+        <li><strong>UAAG 2.0: </strong>(<a href="https://www.w3.org/WAI/intro/uaag">User Agent Accessibility Guidelines Overview</a>):
```

Comment:
This opens up a whole can of worms - there are also UAAG requirements that apply to other topics, like keyboard compatibility - will we add these as well? Also, I feel this is way too technical and specific for the primary target audience - people who are new to web accessibility.